services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -87,12 +87,14 @@ def disable_swagger_doc_generation(
 @pytest.fixture(scope="session")
 def docker_compose_env(default_app_cfg: ConfigDict) -> Iterator[pytest.MonkeyPatch]:
     postgres_cfg = default_app_cfg["db"]["postgres"]
-
+    redis_cfg = default_app_cfg["resource_manager"]["redis"]
     # docker-compose reads these environs
     with pytest.MonkeyPatch().context() as patcher:
         patcher.setenv("TEST_POSTGRES_DB", postgres_cfg["database"])
         patcher.setenv("TEST_POSTGRES_USER", postgres_cfg["user"])
         patcher.setenv("TEST_POSTGRES_PASSWORD", postgres_cfg["password"])
+        # Redis
+        patcher.setenv("TEST_REDIS_PASSWORD", redis_cfg["password"])
```

Comment:
remember my comment before about `get_secret_value`. My guess is that here you are injecting `*******` . 
pls double check.